test/infrastructure/docker/exp/internal/controllers/dockermachinepool_controller.go

```diff
@@ -186,7 +201,11 @@ func (r *DockerMachinePoolReconciler) reconcileNormal(ctx context.Context, clust
 	}
 
 	// Reconcile machines and updates Status.Instances
-	res, err := pool.ReconcileMachines(ctx)
+	managedClusterClient, err := r.tracker.GetClient(ctx, client.ObjectKeyFromObject(cluster))
```

Comment:
```suggestion
	remoteClient, err := r.tracker.GetClient(ctx, client.ObjectKeyFromObject(cluster))
```
nit: to align with how we usually call it in other controllers